internal/driver/modbusclient.go

```diff
@@ -155,14 +155,14 @@ func NewDeviceClient(connectionInfo *ConnectionInfo) (*ModbusClient, error) {
 	client.ModbusType = connectionInfo.Protocol
 	switch client.ModbusType {
 	case ProtocolTCP:
-		client.TCPClientHandler.Address = fmt.Sprintf("%s:%d", connectionInfo.Address, connectionInfo.Port)
+		client.TCPClientHandler = *MODBUS.NewTCPClientHandler(fmt.Sprintf("%s:%d", connectionInfo.Address, connectionInfo.Port))
```

Comment:
should we change to the pointer to those handler in ModbusClient struct?

Reply:
Currently, the get and put command works as before, I don’t think it is necessary to use the pointer. Do you have any concerns?

Reply:
ok, let's keep the chage as miner as possible